ios-swiftui/SherpaOnnx/SherpaOnnx/SherpaOnnxViewModel.swift

```diff
@@ -48,8 +49,19 @@ class SherpaOnnxViewModel: ObservableObject {
         self.subtitles = self.results
     }
 
+    func setupAudioSession(){
+        audioSession = AVAudioSession.sharedInstance()
+        do{
+            try audioSession.setCategory(.playAndRecord, mode: .default, options: [.defaultToSpeaker])
+            try audioSession.setActive(true)
+        }catch{
```

Comment:
Can you use xcode to reformat your code?

For instance, there should exist a space between `do` and `{`.

It is `do {` , not `do{`.